e2e/schematics/application.test.ts

```diff
@@ -1,6 +1,21 @@
-import { newApp, newLib, newProject, runCLI, updateFile } from '../utils';
+import {
+  newApp,
+  newLib,
+  newProject,
+  runCLI,
+  updateFile,
+  cleanup
+} from '../utils';
 
 describe('Nrwl Workspace', () => {
+  beforeAll(() => {
+    cleanup();
```

Comment:
why is this required?

`newProject()` will always do cleanup, right?  And every test invokes it.

Reply:
Its not. I'll remove all of the occurances